rs/bitcoin/adapter/tests/adapter_test.rs

```diff
@@ -503,6 +539,130 @@ fn test_receives_blocks() {
     assert_eq!(blocks.len(), 150);
 }
 
+// Checks that the adapter disconnects from the clients when it becomes idle.
+#[test]
+fn test_adapter_disconnects_when_idle() {
+    let logger = no_op_logger();
+
+    let bitcoind1 = get_default_bitcoind();
+    let client1 = Client::new(
```

Comment:
Similar to the comment below, I don't think 3 bitcoin peers are needed here. I believe 1 is sufficient for the test

Reply:
Done